packages/widget_toolkit_pin/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.1.0]
+- Improvements to error handling
+### Breaking changes:
+- Update `PinCodeService.verifyPinCode` to return a `Future<dynamic>` instead of a `bool`
```

Comment:
Please add information for the other callback-related breaking changes

Reply:
Updated! Let me know if I missed something.